internal/storages/directory/directory.go

```diff
@@ -188,3 +188,10 @@ func (d *Storage) SubStorage(dp string, relative bool) storages.Storager {
 		fileMode: d.fileMode,
 	}
 }
+
+func fixPrefix(prefix string) string {
```

Comment:
This code is doubled. Could you move this function into let's say `internal/utils/config` and make this function public (uppercase first letter)